content/en/continuous_integration/pipelines/teamcity.md

```diff
@@ -0,0 +1,104 @@
+2. If you don't already have one, add a [TeamCity composite build][6] as the last build of the build chain. This build must have
+a dependency on the current last build of the chain and no other builds depending on it. Build chains not having a
+composite build as their last build are ignored by the plugin. Below is an example of an expected build chain,
+in which `Aggregating Results` is the last composite build:
+{{< img src="ci/teamcity_build_chain.png" alt="TeamCity build chain with composite build at the end" style="width:90%;">}}
+The final composite build must be properly configured in terms of version control settings, with
+the VCS Root attached and the [VCS Trigger][13] configured.
+3. The following configuration parameters need to be present for TeamCity projects:
+   * `datadog.ci.api.key`: Your [Datadog API Key][2].
+   * `datadog.ci.site`: {{< region-param key="dd_site" code="true" >}}.
+   * `datadog.ci.enabled`: `true` (`false`
+   can be used to disable the plugin for a specific project).
+
+   These configuration parameters can be added either to TeamCity subprojects
+   or to the [TeamCity Root Project][10]. When added to the Root project, they are propagated
+   to all its subprojects. For example, to enable the plugin for all projects, add `datadog.ci.enabled` with the
+   value `true` to the Root Project. More information on defining configuration parameters
+   is available in the [TeamCity Project Hierarchy][9] documentation.
+4. To enable the plugin, click on **Enable uploaded plugins** in the **Administration** -> **Plugins** page.
+Alternatively, restart the TeamCity server.
+
+## Visualize pipeline data in Datadog
+
+View your data on the [Pipelines][3] and [Pipeline Executions][4] pages after the pipelines finish.
+
+**Note**: The Pipelines page only shows data for the [default branch][12] of each repository.
+
+## Troubleshooting
+
+All the logs generated by the Datadog CI Integration plugin are stored inside the `teamcity-server.log` file and can be
+accessed from the TeamCity Server by going to **Administration** -> **Diagnostic** -> **Server Logs**.
+Check these logs to get additional context on any issues with the plugin.
+
+## Git user information
+
+The plugin retrieves the Git author name and email based on the [TeamCity username style][7].
+Datadog recommends using either **Author Name and Email** or **Author Email** username styles, as they
+provide information about the user email. When one of the other username styles is used (**UserId** or **Author Name**),
+the plugin automatically generates an email for the user by appending `@Teamcity` to the username.
+For example, if the **UserId** username style is used and the git author username is `john.doe`,
+the plugin generates `john.doe@Teamcity` as the Git author email. The username style is defined for [VCS Roots][11],
+and can be modified in the VCS Root settings.
+
+<div class="alert alert-info"><strong>Note:</strong> The git author email is used for
```

Comment:
```suggestion
<div class="alert alert-danger"><strong>Note:</strong> The Git author email is used for
```

Reply:
Optional suggestion to update this to be red and more eye-catching because of the potential billing implications.

Reply:
Makes sense to me, I'll double check that visually is still ok